content/pt/docs/reference/access-authn-authz/bootstrap-tokens.md

```diff
@@ -0,0 +1,169 @@
+* `usage-bootstrap-signing` indica que o token pode ser utilizado para assinar o ConfigMap `cluster-info` como descrito abaixo.
+ 
+O campo `expiration` controla a expiração do token. Tokens expirados são
+rejeitados quando usados para autenticação e ignorados durante assinatura de ConfigMaps.
+O valor de expiração é codificado como um tempo absoluto UTC utilizando a RFC3339. Para automaticamente
+remover tokens expirados basta habilitar o controlador `tokencleaner`.
+ 
+## Gerenciamento de tokens com kubeadm
+ 
+Você pode usar a ferramenta `kubeadm` para gerenciar tokens em um cluster. Veja [documentação de tokens kubeadm](/docs/reference/setup-tools/kubeadm/kubeadm-token/) para mais detalhes.
+ 
+## Assinatura de ConfigMap
+ 
+Além de autenticação, os tokens podem ser utilizados para assinar um ConfigMap. Isto pode
+ser utilizado em estágio inicial do processo de inicialização de um cluster, antes que o cliente confie
+no servidor de API. O Configmap assinado pode ser autenticado por um token compartilhado.
+ 
+Habilite a assinatura de ConfigMap ao habilitar o controlador `bootstrapsigner` no Gerenciador de Controle.
+ 
+```
+--controllers=*,bootstrapsigner
+```
+O ConfigMap assinado é o `cluster-info` no namespace `kube-public`.
+No fluxo típico, um cliente lê o ConfigMap enquanto ainda não autenticado
+e ignora os erros da camada de transporte seguro (TLS).
+Ele então valida o conteúdo do ConfigMap ao verificar a assinatura contida no ConfigMap.
+ 
+O ConfigMap pode se parecer com o exemplo abaixo:
+ 
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+ name: cluster-info
+ namespace: kube-public
+data:
+ jws-kubeconfig-07401b: eyJhbGciOiJIUzI1NiIsImtpZCI6IjA3NDAxYiJ9..tYEfbo6zDNo40MQE07aZcQX2m3EB2rO3NuXtxVMYm9U
+ kubeconfig: |
+   apiVersion: v1
+   clusters:
+   - cluster:
+       certificate-authority-data: <really long certificate data>
+       server: https://10.138.0.2:6443
+     name: ""
+   contexts: []
+   current-context: ""
+   kind: Config
+   preferences: {}
+   users: []
+```
+ 
+O membro `kubeconfig` do ConfigMap é um arquivo de configuração contendo somente
+as informações do cluster preenchidas. A informação chave sendo comunicada aqui
+está em `certificate-authority-data`. Isto poderá ser expandido no futuro.
+ 
+A assinatura é feita utilizando-se assinatura JWS em modo "separado". Para validar
+a assinatura, o usuário deve codificar o conteúdo do `kubeconfig` de acordo com as regras do JWS
+(codificando em base64 e descartando qualquer `=` ao final). O conteúdo codificado
+e então usado para formar um JWS inteiro, inserindo-o entre os 2 pontos. Você pode
+verificar o JWS utilizando o esquema `HS256` (HMAC-SHA256) com o token completo
+(por exemplo: `07401b.f395accd246ae52d`) como o segredo compartilhado. Usuários _devem_
+verificar que o algoritmo HS256 (que é um método de assinatura simétrica) está sendo utilizado.
+ 
+ 
+{{< aviso >}}
```

Comment:
here, please keep the original 'warning' shortcode, it's used to generate the warning box :)